### PR TITLE
First clone and initialize folders

### PR DESCRIPTION
**Pull request by kimgayeon**
landing pages ( place image)
trending classes - image scroller
calendar

### CODE REVIEW

File: Android/.idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <Objective-C-extensions>
+      <file>
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Import" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Macro" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Typedef" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Enum" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Constant" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Global" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="Struct" />
+        <option name="com.jetbrains.cidr.lang.util.OCDeclarationKind" value="FunctionPredecl" />
```
Comment (ethkim) [inline Android/.idea/codeStyles/Project.xml]: Make the lines shorter to fit within the github's codereview window (I think it was 80 chars or 100 chars, I forget).

File: Android/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    defaultConfig {
+        applicationId "com.example.yello.common"
+        minSdkVersion 27
+        targetSdkVersion 27
+        versionCode 1
+        versionName "1.0"
```
Comment (ethkim) [inline Android/app/build.gradle]: Alphabetically sort all list of arguments - it makes it easier to spot when reading the code.

File: Android/app/src/main/java/com/example/yello/common/Adapter.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class Adapter extends PagerAdapter {
+
+    private Context context;
+    private LayoutInflater inflater;
+    private Integer [] images = {R.drawable.coffee, R.drawable.cooking, R.drawable.wine}; // image of classes
+    private String [] classes = {"Coffee", "Cooking", "Wine tasting"};
```
Comment (ethkim) [inline Android/app/src/main/java/com/example/yello/common/Adapter.java]: These should be supplied from the server, correct? If this is a mock data, please leave a comment & add a TODO to remove it later when the backend responses are available.

File: Android/app/src/main/java/com/example/yello/common/SecondActivity.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.DatePicker;
+import android.widget.GridView;
+import android.widget.Toast;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+
+public class SecondActivity extends AppCompatActivity {
```
Comment (ethkim) [inline Android/app/src/main/java/com/example/yello/common/SecondActivity.java]: What does "SecondActivity" mean? Let's choose a better name.

File: Android/app/src/main/java/com/example/yello/common/SecondActivity.java [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    Adapter adapter;
+    ViewPager classViewPager;
+    DatePicker datePicker;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_second);
+
+        //Intent in = getIntent();
```
Comment (ethkim) [inline Android/app/src/main/java/com/example/yello/common/SecondActivity.java]: Remove all commented out code.

File: Android/app/src/main/java/com/example/yello/common/Adapter.java (hunk 3 above)
Comment (kimgayeon) [reply Android/app/src/main/java/com/example/yello/common/Adapter.java]: Yeah, Those should be changed later. I will add comment.

File: Android/.idea/codeStyles/Project.xml (hunk 1 above)
Comment (kimgayeon) [reply Android/.idea/codeStyles/Project.xml]: Okay. I changed all of them.

File: Android/app/src/main/java/com/example/yello/common/SecondActivity.java (hunk 4 above)
Comment (kimgayeon) [reply Android/app/src/main/java/com/example/yello/common/SecondActivity.java]: I changed it to TrendingClassActivity.
Also, I changed MainActivity to PlaceActivty.

File: Android/app/build.gradle [hunk 6]
```diff
@@ -6,9 +6,9 @@ android {
         applicationId "com.example.yello.common"
         minSdkVersion 27
         targetSdkVersion 27
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```
Comment (ethkim) [inline Android/app/build.gradle]: # of chars.

Comment (kimgayeon) [reply Android/app/build.gradle]: Sorry.... I confused that. I modified all of them.

Comment (kimgayeon) [reply Android/app/build.gradle]: But I can't modify gradlew file in Window... So I modifed files except gradlew.

**Comment by kimgayeon** [comment]
started to make ReservationActivity and added time selector

**Comment by kimgayeon** [comment]
add num of ticket selector

**Comment by kimgayeon** [comment]
add notification message and merge

**Pull request merged**